CompanyEmployees.Core.Domain/Repositories/IAttendanceRepository.cs

```diff
@@ -0,0 +1,13 @@
+using CompanyEmployees.Core.Domain.Entities;
+
+namespace CompanyEmployees.Core.Domain.Repositories;
+
```

Comment:
Remove the semicolon after the namespace declaration. In C#, the `namespace` keyword should be followed by a body (enclosed in braces), not a semicolon.

**Suggested change**
```suggestion
namespace CompanyEmployees.Core.Domain.Repositories
{
```